client/src/components/PartenerMedia.js

```diff
@@ -82,53 +84,69 @@ function YouTubeButton(text1, text2) {
 }
 
 function PartenerMedia() {
+  const ref = useRef(null);
+  const isInView = useInView(ref, { once: true });
+  const mainControls = useAnimation();
+  const slideControls = useAnimation();
+
+  useEffect(() => {
+    if (isInView) {
+      mainControls.start("visible");
+      slideControls.start("visible");
+    }
+  }, [isInView, mainControls, slideControls]);
   return (
     <>
-      <center className="pb-16">
-        <h1 className="text-3xl font-bold text-white">
-          ALLSTARS GAMING: Partener Media
-        </h1>
-      </center>
-      <center className="pb-16">
-        <a id="backToTop" href="#">
-          <h2
-            className="text-2xl font-neuropolxbold underline"
-            style={{ color: "#78e800", fontWeight: "bold" }}
-          >
-            CUPRINS
-          </h2>
-        </a>
-      </center>
-      <center className="pb-16 transform transition-transform duration-600 hover:scale-110">
-        <a href="https://allstars-gaming.ro/">
-          <img src="https://allstars-gaming.ro/wp-content/uploads/2022/07/logo-allstars-mic2.png"></img>
-        </a>
-      </center>
+      <motion.div
+        ref={ref}
+        id="projects"
+        variants={{
+          hidden: { opacity: 0, y: 75 },
+          visible: { opacity: 1, y: 0 },
+        }}
+        initial="hidden"
+        animate={mainControls}
+        transition={{ duration: 0.5, delay: 0.25 }}
+      >
+        <center className="pb-16">
+          <h1 className="text-3xl font-bold text-white">
+            ALLSTARS GAMING: Partener Media
+          </h1>
+        </center>
+        <center className="pb-16 transform transition-transform duration-600 hover:scale-110">
+          <a href="https://allstars-gaming.ro/">
```

Comment:
try not to center with the center tag. here is an example : https://tailwindcss.com/docs/align-content